topic-controller/resources/openshift-template.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: strimzi-topic-controller
+  labels:
+    app: strimzi
+---
+apiVersion: v1
+kind: ClusterRole
+metadata:
+  name: strimzi-topic-controller-role
+  labels:
+    app: strimzi
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - patch
+  - update
```

Comment:
Doesn't the topic controller need the "delete" verb as well ? Or there is no use case where a topic ConfigMap should be deleted ?

Reply:
A followup PR will do this.